### PR TITLE
You can no longer equip pet collars on humans

### PR DESCRIPTION
**Pull request by optimumtact**
This just heads the drama all off at the pass and should be a non issue as it doesn't break the supposed functionality of the item.

:cl: oranges
del: Pet collars no longer fit on humans
/:cl:

### CODE REVIEW

**Comment by RemieRichards** [comment]
You've lost your way fruitboi turn back.

**Comment by RogueSteampunker** [comment]
But how else am I supposed to punish assistants that break into my office as a Cap? I don't want to cut off ALL their limbs.

**Comment by optimumtact** [comment]
This should be a non issue if the collars are only supposedly used for their intended reasons, so why would anyone complain?

**Comment by RemieRichards** [comment]
Their intended use is implied by them being ```/clothing/neck```

**Comment by Iamgoofball** [comment]
we should go the extra mile and just make the collars start on the animals too honestly

**Comment by RemieRichards** [comment]
image

**Comment by optimumtact** [comment]
@GunHog care to explain what use you have for equipping these on humans?

**Comment by optimumtact** [comment]
@RemieRichards but why would you you, weirdo

**Comment by The-clown-known-as-Pepe** [comment]
@optimumtact remove them from humans, restrict collars to catgirls only.

**Comment by RemieRichards** [comment]
But it's still clearly part of the intention~ change up your argument.

**Comment by GunHog** [comment]
@optimumtact Simply because I find it amusing. I have not used one myself as I do not have usually have access to one.

**Comment by optimumtact** [comment]
I guess we have different senses of humour then

**Comment by Militaires** [comment]
remie you should swear at kor now that he's headcoder

**Comment by Militaires** [comment]
it'll get oranges banned i promise

**Comment by RemieRichards** [comment]
no need, I've called him a pushover enough times lmao.

**Comment by optimumtact** [comment]
Can we keep this just to this topic? It's going to get a lot of replies anyway probably

**Comment by Firecage** [comment]
I disagree with this personally. I frequently wear this since it looks good as a clothing option.

If people are using it for ERP or using it for other creepy things which disrupts the server, then it should be easy for administration to deal with said individuals.

(Also, it is fun to stuff into down disposal, put on his collar, and then buckle myself to Ians bed. One HoP ran out in horror.)

**Comment by ShizCalev** [comment]
Boooooooooooo 👎

I usually put the collar on as the HOP along with the Ian shirt. Stop ruining my Ian cosplay RP experience

**Comment by Firecage** [comment]
Well, at least you didn't choose the extreme option for collars. Which would have been that there's a X% chance for a Cursed Collar to spawn instead, which when worn, changes the human into a corgi.

**Comment by optimumtact** [comment]
Sounds like a lavaland loot

**Comment by Firecage** [comment]
My god...that's genius...

**Comment by Iamgoofball** [comment]
https://file.house/bqGN.png

this is creepy

this is really, really creepy

**Comment by RogueSteampunker** [comment]
So, we remove a feature from an item so it can't be wearable.

And to replace it, it can be used to rename pets. You know, those things that are disregarded at best, and only paid attention to when they're shoved into the washing machines at worst.

Seems about right.

**Comment by Slignerd** [comment]
> This just heads the drama

\>Thing exists
\>You hate the thing, so you start drama about it
\>Use the fact that there's drama now as an argument to remove thing

Can we stop used drama as an argument to remove things, considering that just leads to self-induced drama for the sake of pushing changes to things you don't like?

**Comment by optimumtact** [comment]
@RogueSteampunker it's not really removing a feature

The primary feature of this item is you can put it on pets, which is still completely possible and it's intended use.

**Comment by RogueSteampunker** [comment]
@optimumtact But it's a feature that's never used, is the problem. It seems worthless to argue the "intended" purpose, when that purpose is rarely if ever utilized.

**Comment by Slignerd** [comment]
Pretty much everything about the implementation indicates that you're supposed to be able to wear it as a human mob. So if you want to argue about "intended purpose", then pet collars being wearable was part of it.

**Comment by optimumtact** [comment]
But that would imply that the only reason this item exists is to be worn by a sub group of people who wear it for fetish reasons in which case it would need to be removed permanently anyway.

**Comment by Slignerd** [comment]
> it's a space station game about spacemen in space

Which happens to be filled with memes, and some weirdos wearing pet collars and getting ridiculed for it is one of these memes.

**Comment by RogueSteampunker** [comment]
>Assumes entire existence is for fetishes
>Implies that for reason to be removed

Oh, that's just lovely. I forgot the catgirl argument could be applied to anything that could be associated with "furries"

**Comment by RogueSteampunker** [comment]
So what does that say for the maid outfit? The chicken/bee suit? Are they just there for no reason, because they aren't inherently useful in combat?

Besides, forcing people to wear it and handcuffing them to shit like Ian's bed is always funny when they break into the office.

**Comment by optimumtact** [comment]
Those items at least have a more cohesive aesthetic, i.e you wear a maid outfit and you look like a maid, wear a bee suit and you look like a bee.

The collar doesn't meet that same level change, you put it on and it adds a little bar across your throat.

As to your last point, the only reason it's "funny" is the whole power dynamic of the pet/owner relationship

**Comment by RogueSteampunker** [comment]
More because assistants start threatening my life because I didn't just let them steal all-access, but that's besides the point.

And if it's such a small change, why does it matter to anyone besides the minuscule minority of people who believe anything animal-related suddenly means fetish fuel?

**Comment by optimumtact** [comment]
That argument doesn't make sense because it can be applied the exact other way.

IF this is such a small change, why does it matter to anyone besides the minuscule minority of people who wear animal specific items as a human.

That basically cancels out either way.

There has to be some reason to keep this feature in that outweighs the harm of having it attract the kind of people who think wearing it is a good idea.

**Comment by RogueSteampunker** [comment]
As it was stated above, removing it would prove that causing worthless and mindless drama over a subject is a genuine means of getting stuff you don't agree with removed. Keeping it merely means a feature that was generally unoffensive isn't removed because a few people threw a fit.

And as for keeping it, I'd argue that the alternative would be making an item near useless and generally disregarded. There's no inherent harm in keeping it wearable, and anyone that does get really creepy with it is usually in violation of the rules anyway. So removing it just seems like a general waste of an item, and the validation that making enough of a hissyfit over something you don't like can get it removed.

**Comment by sawrge** [comment]
*hordes of furries come pouring out of the woodwork to play a terrible byond game full of saucy angry children and nerds because they can equipt a collar*

**Comment by optimumtact** [comment]
It's easy for you guys to say, you don't have to deal with goof and the team's constant PR's

**Comment by sawrge** [comment]
Sounds like you should just gitban the dude trolling the entire codebase for fun and making it his passtime to stir up as much shit as possible

**Comment by Iamgoofball** [comment]
Because having differing opinions is trolling now.

**Comment by optimumtact** [comment]
@RogueSteampunker we disagree about the levels of harm in making it wearable but that's just opinions that we both have

**Comment by Iamgoofball** [comment]
@sawrge idk man really seems like the number of unironic furries really jumped up after catpeople got added

**Comment by sawrge** [comment]
I don't really care about this change but "it attracts the wrong type of people" is such a fucking goofy argument especially for minor aesthetic options especially when you consider who the previous hosts were, half the fucking admins, the main playerbase when the community was much smaller, etc etc

**Comment by RogueSteampunker** [comment]
@optimumtact And I understand that. Which is why I talked of the lack of use it'll see if it's removed. Because it will not see much to any use the second it's literally just made a tool to rename the pets no one looks at. Whereas in it's current state, it's at least usable as a wearable item.

**Comment by Slignerd** [comment]
>the number of unironic furries really jumped up after catpeople got added

We don't really have furries? We have catgirls, though you did make it clear numerous times that you just can't tell the difference.

**Comment by duncathan** [comment]
on average, the furries i know are much easier to interact with than goofball often is so idk if they're even the wrong kind of players to attract tbh

**Comment by Slignerd** [comment]
All the embarrassing furry stuff generally stays at Paradise anyways.

**Comment by Iamgoofball** [comment]
catpeople are closet furries

we all know it

everyone knows it

**Comment by Supermichael777** [comment]
Some players are toxic
some players are furries (weirdos)
logically your going to end up with some toxic furries

**Comment by RogueSteampunker** [comment]
@Iamgoofball >All catgirls are closet furries
>Insinuates that the "nonironic furries" aren't all that public.

Way to go, Goofy boy. You made another fallacy in your own arguments.

**Comment by Slignerd** [comment]
As said, goofball just can't tell the difference, so his judgment on how many "furries" there are simply isn't reliable.

> really seems like the number of unironic furries really jumped up after catpeople got added

What he means here is that the number of catpeople jumped up after catpeople got added, which kinda goes without saying tbh.

**Comment by FantasticFwoosh** [comment]
> @Firecage : which when worn, changes the human into a corgi 

And if you take it off they turn back into a human again. Even i can't really complain about that.

Without actually mentioning **_them_**, its like steelpoint PR's and Ex-CM gun enthusiasts, it attracted attention to people exiled or otherwise shunned from their own communities for being toxic.

**Comment by KorPhaeron** [comment]
@FantasticFwoosh 

Didn't you quit

**Comment by FantasticFwoosh** [comment]
Yeah but no borders on github, like your immigration policy putting basil on the hub attracting drama minorities of banned problem players that fuel PR's like this. 

Its in my interests to remain vocal if i at all want to play on the downstream servers or anything on former TG code im familiar with. Im not strictly locked in a Bloc.

**Comment by sawrge** [comment]
@FantasticFwoosh 
Didn't you quit

**Comment by Aliannera** [comment]
@FantasticFwoosh

Didn’t you quit

**Comment by RogueSteampunker** [comment]
@FantasticFwoosh 
Didn't you quit

**Comment by Mercenaryblue** [comment]
> (Also, it is fun to stuff Ian down disposal, put on his collar, and then buckle myself to Ian's bed. One HoP ran out in horror.)

As a regular clown, this is both frightening and hilarious.

**Comment by Slignerd** [comment]
>Yeah but no borders on github, like your immigration policy putting basil on the hub attracting drama minorities of banned problem players that fuel PR's like this.

lol

**Comment by Bawhoppen** [comment]
oranges will selfmerge this and then the dictatorship will be complete

**Comment by kevinz000** [comment]
Lmao

**Comment by Slignerd** [comment]
>oranges will selfmerge this and then the dictatorship will be complete

Nah, lzimann's gonna merge it instead.

Conveniently enough, there's two of them - that way they can just push their personal agenda without any regard for the playerbase.

**Comment by optimumtact** [comment]
Removed the fwoosh derail and bullshit bait about selfmerging/others merging

**Comment by Bawhoppen** [comment]
Oh come on oranges it's a joke I've said a thousand times, not bait

Stop being revisionist and deleting posts like you constantly do now

**Comment by Slignerd** [comment]
I exposed coderbus conspiracy, they're onto me now.

**Comment by Qbopper** [comment]
if this is really going to be such a dumb back and forth meme argument why don't we escalate to full meme and player poll "should pet collars be wearable by humans"

**Comment by RogueSteampunker** [comment]
@Qbopper That seems like the logical solution to all of this

**Comment by Cyberboss** [comment]
https://atlantaned.space/newSS13tools/polls.php?poll=185

**Comment by KorPhaeron** [comment]
@optimumtact 

Is it attracting the wrong sorts of people though? Cheridan is the one who added it, and Shizcalev/Remie/GunHog are among those who are defending it, and I think they've all had an enormous positive impact on the server

The ones supporting its removal include people from downstreams who will never play here anyway, a guy you threatened to gitban earlier, and fwoosh

I agree it's an out of place non feature and I won't care if it's removed from the game, I just think caving to shitposting campaigns or worrying about "undesirables" (when the weebs, fetishists, furries, etc have been helping to build this codebase since r4407) are poor rationales for removal

**Comment by KorPhaeron** [comment]
@Cyberboss oh god why

**Comment by Qbopper** [comment]
wow

this is probably the first time a suggestion of mine was taken seriously and put into action instantly

**Comment by ChangelingRain** [comment]
I think we should either:

1. Merge this and gitban goof.
2. Close this and leave goof unbanned.

I believe this is a fair compromise?

**Comment by Cyberboss** [comment]
1xiwo8 1

**Comment by Bawhoppen** [comment]
Yea I think that's an idea we can all get behind

**Comment by Qbopper** [comment]
@ChangelingRain why can't we have him being gitbanned for both options :^)

**Comment by RogueSteampunker** [comment]
@Qbopper I like the way you think

**Comment by Iamgoofball** [comment]
cool, gitbanning for, once again, disagreeing with you

i love it

really classy, /tg/

good to know the 4+ years of development i've put into this project is being respected

**Comment by Qbopper** [comment]
no not for disagreeing with me just because i don't like you

**Comment by Iamgoofball** [comment]
are you really this petty

**Comment by Bawhoppen** [comment]
Hey yea Goof, I'd just like to remind you, that considering what you've done in the past, the only reason you HAVEN'T been gitbanned yet, is BECAUSE of your long history of development.

**Comment by Iamgoofball** [comment]
@Bawhoppen quite frankly I don't think I need to take this crap from maintainers like @changelingrain who literally abused their powers to sabotage Oldman Robustin's PRs. 

they, of all people, do not have any right to dictate bans

**Comment by Qbopper** [comment]
i went fishing for goof and i caught one thiiiiiis big

**Comment by Iamgoofball** [comment]
fug

**Comment by Grazyn** [comment]
It's the closest thing we have to a choker, some players like the appearance of a choker on their female character, so if you want to remove this at least add an actual choker item.

**Comment by Militaires** [comment]
this pr is good

**Comment by KieranCat** [comment]
Why are goof memes still a thing

**Comment by FantasticFwoosh** [comment]
@KorPhaeron down-streams without PR bots have to go to the effort to remove all this content manually each time they download, which can be a utter pain in the ass as not everyone downstream evidently by your words reads off the same page about it. Its more a sphere of wider responsibility for "**When in Rome do as Romans do**" in that people who don't want to play don't engage, and try to observe the rules & customs of a server (like a country) when you enter it.

Yes those persons have contributed to 'this' community but pushing PR's only relevant here elsewhere doesn't help. Worst case scenario is that they will just stop using TG code altogether.

**Comment by Slignerd** [comment]
Why does Fwoosh continue to make horrible non-arguments?

**Comment by RogueSteampunker** [comment]
So, as the first 24 hours comes to a close, and the threat of an auto-merge looms ever closer, lets have a recap.

Reasons to keep it:
Item becomes worthless if ability to wear it is removed.

Validates trolls that bullying maintainers is all it takes to get what you want if it's removed.

It is a minuscule feature in it of itself, and is generally unoffensive.

Reasons to remove it:
The maintainers don't want to dole out github bans, but don't want to deal with the trolls either.

An apparent pet collar's wearability causes furries to flock here in droves, despite no confirmation of this.

It's "fetish wear", since there's no combat reason to wear it, despite the maid suit being in the same boat, only much more visible.

**Comment by Grazyn** [comment]
I think it's counterintuitive to have a neck slot and an obvious neck item that can't be worn in that slot. It will just diminish the sense of interactivity from the eyes of new players, like in those games where you expect something to work logically but it doesn't due to oversight or arbitrary game design. It reduces freedom in a game where freedom to do anything you want with your environment should be the main appeal.

**Comment by Slignerd** [comment]
Okay, why exactly did @optimumtact remove the "do not merge" tag from his own PR?

**Comment by duncathan** [comment]
For the sake of clarity I disagree with this pr

**Comment by KorPhaeron** [comment]
@duncathan Another furry revealed

**Comment by duncathan** [comment]
I'm just an ally

**Comment by Bawhoppen** [comment]
furry sympathizer more like

**Comment by KorPhaeron** [comment]
>I think it's counterintuitive to have a neck slot and an obvious neck item that can't be worn in that slot. It will just diminish the sense of interactivity, like in those games where you expect something to work logically but it doesn't due to oversight or arbitrary game design. It reduces freedom in a game where freedom to do anything you want with your environment should be the main appeal.

This is a good argument

**Comment by Cyberboss** [comment]
Also, associating collars with erp and then, by extension, furries isn't logical imo. Pick one or the other

**Comment by duncathan** [comment]
As I said before the furries I know are easier to deal with than goofball so I really don't think they're that bad

**Comment by KorPhaeron** [comment]
3 hours till leoz merges this and the player poll comes back "keep this" and someone opens a revert and we have an actual maintainer vote on the ethics of bdsm in atmos simulators

**Comment by Bawhoppen** [comment]
Isn't it a player issue anyways, rather than a code issue?

**Comment by sawrge** [comment]
As kor appointed design lead I am closing this pr

**Comment by duncathan** [comment]
what does that mean

**Comment by sawrge** [comment]
Help it's not working

**Comment by ChangelingRain** [comment]
of course it's not I'm the kor appointed design lead

**Comment by duncathan** [comment]
as the kor appointed design lead I support this closure

**Comment by The-clown-known-as-Pepe** [comment]
as the kor appointed design lead I blorble

**Comment by lzimann** [comment]
repath it so it's no longer a neck slot item(which makes no sense in being one)

**Comment by Shihrt** [comment]
except it does

**Comment by lzimann** [comment]
it really doesn't have any reason to be a neck item though

**Comment by The-clown-known-as-Pepe** [comment]
@lzimann then what it should be, leg item?, hand item?

the obvious use of a collar is to put it on a neck.

**Comment by jughu** [comment]
this station is a hospital for the insane

**Comment by Iamgoofball** [comment]
not a clothing item at all, and turned into a generic object

**Comment by Shihrt** [comment]
the ideal ss13, no fetishes
fetishfree

**Comment by duncathan** [comment]
my fetish is atmospheric simulation

**Comment by Grazyn** [comment]
>see collar
>notice you have a neck slot
>"geez I wonder if this game is so carefully detailed that I can put this around my neck"
>"you can't equip that!"
>"I guess not"

I mean, I would be fine with it at least giving a message like "the collar is too tight", to at least acknowledge that, yes, it is indeed a neck item and yes, we thought of someone trying to wearing it.

**Comment by optimumtact** [comment]
If you'd prefer I can make it deal you massive amounts of brain damage instead of preventing it equipping on you

**Comment by jughu** [comment]
If your first thought is to put a collar on your neck you have some serious problems

**Comment by sawrge** [comment]
Jughu that's typically where they go

**Comment by optimumtact** [comment]
no they go on the neck of the shirt dummy

**Comment by The-clown-known-as-Pepe** [comment]
Where does the collar go

A. Neck
B. Ankle
C. Wrist
D. Waist

74% of maintainers don't know the correct answer!

Are you one of them?

**Comment by Militaires** [comment]
keep it but have it make you retarded with time

**Comment by Militaires** [comment]
there you go, freedom preserved, emergent gameplay.

**Comment by Slignerd** [comment]
\> removing "do not merge" tag from, and re-opening own PRs
\> editing and deleting people's messages

**Comment by sawrge** [comment]
Even if she is misguided sometimes she is a very good maintainer

**Comment by sawrge** [comment]
And I love her.

**Comment by Qustinnus** [comment]
i googled "person wearing pet collar" and to my surprise i got a bunch of fetishist stuff. its almost like petplay really is a fetish.

**Comment by Qustinnus** [comment]
That works with catgirls too, by the way. Except with catgirls most of the pictures are pictures of questionably aged anime catgirls in lingerie.

**Review by lzimann — CHANGES_REQUESTED** [review]
if you want to prevent humans from equipping, this needs a repathing to stop being a clothing item

**Comment by sawrge** [comment]
Google a clue

**Comment by Repukan** [comment]
Add me to the screen cap

**Comment by XDTM** [comment]
https://atlantaned.space/newSS13tools/polls.php?poll=185

wasn't goof the one that wanted to change things based on playerbase votes?

**Comment by Ausops** [comment]
oranges is trying to bring some much needed vitamin C(lean this codebase up from degeneracy), let him do so

**Comment by lzimann** [comment]
why the fuck was a poll even made for this

**Comment by RemieRichards** [comment]
@XDTM as has been said a multitude of times, he only cares about polls when they agree with him.

**Comment by Repukan** [comment]
Didn't you leave Remie?

**Comment by RemieRichards** [comment]
I float around, I did a PR for hippie, stuff like that.
I'm way less active but I like to poke fun in places still (drama is drama)

**Comment by ChangelingRain** [comment]
why is oranges reopening his own closed prs
shameful

**Comment by Repukan** [comment]
Teach me how to code remi pls

**Comment by lzimann** [comment]
ok fine i do it then

**Comment by RemieRichards** [comment]
@Repukan I mean, if you're serious I can teach you DM.

**Comment by Thunder12345** [comment]
@lzimann I told him to repath them before this was ever opened and he threw an autistic shitfit about it still being clothing because not all clothes are worn by humans or something

**Comment by KorPhaeron** [comment]
Why can't you guys be fighting over something that matters

**Comment by Iamgoofball** [comment]
Because you haven't demaintainered a person who literally sabotaged PRs and won't admit to it, even going as far as evading bans to get around it, and her and her lackey just screams to gitban me every time it's brought up. Almost like they want to censor me.

**Comment by lzimann** [comment]
isn't it funny how a person that has done that is still more relevant and important to the community than you?

**Comment by KorPhaeron** [comment]
oranges is the very opposite of a joan lackey

**Comment by Bawhoppen** [comment]
Shouldn't all these comments be deleted by oranges for offtopic? Or does he pick and choose?

**Comment by KorPhaeron** [comment]
@Bawhoppen I'm sure they will be eventually be he does not monitor this PR 24 horus a day

**Comment by Ausops** [comment]
make collars decapitate humans unless they are emagged or emp'd

**Comment by Thunder12345** [comment]
@KorPhaeron But balance changes and major new features are hard. Fighting over a stupid collar is easy.

**Comment by Slignerd** [comment]
It's convenient how oranges and lzimann have eachother's backs so that they completely disregard the playerbase, together.

**Comment by Repukan** [comment]
So? 

It's not a democracy and the playerbase cannot imply soft power over the voluntary coders.

**Comment by Militaires** [comment]
@Slignerd kor and joan do this too it's not a new thing. when i tried to shit on their lawn about it they said it's "maintainers having each other's backs". fug

**Comment by duncathan** [comment]
I'm against this PR, but because another maintainer opened it I have the respect to humour it until a consensus is reached. Believe it or not, the maintainers actually are a team and we try to work together to make this game the best it can be. If oranges or any other maintainer thinks that gitbanning someone is gonna improve the quality of our working environment, he probably has a pretty good reason. It's a really simple example of people working together and supporting their teammates. In what world does that make us "lackeys?" Lmao.

**Comment by sawrge** [comment]
@duncathan he's baiting just don't reply

**Comment by RemieRichards** [comment]
Lol, team my ass.

**Comment by Militaires** [comment]
holy fuck this turned into a seinfeld sitcom with that remie take

**Comment by Militaires** [comment]
god oranges is such a legend

**Comment by Iamgoofball** [comment]
oranges isn't the joan lackey, lzimann is

**Comment by duncathan** [comment]
anyone who thinks I'm not Joan's biggest lackey is sorely mistaken

**Comment by Iamgoofball** [comment]
the poster below is a joan lackey

say we know it's you

**Comment by XDTM** [comment]
damn how did he find me out

**Comment by Bawhoppen** [comment]
XDTM...
more like 
XSI

**Comment by Slignerd** [comment]
real talk, does Joan even have lackeys anymore

**Pull request closed without merging**
